style.css

```diff
@@ -0,0 +1,633 @@
+  width: 55%;
+}
+
+.article-footer-btn button {
+  background-color: #567b95;
+  border: none;
+  border-radius: 3px;
+  color: white;
+  width: 100%;
+  height: 35px;
+  font-weight: bold;
+}
+
+.article-footer-icons {
+  display: flex;
+  justify-content: flex-end;
+  width: 45%;
+}
+
+.article-footer-icons span {
+  font-size: 27px;
+  padding: 0 10px;
+}
+
+.article-footer-icons a {
+  text-decoration: none;
+  color: #837d7d;
+}
+
+.article-footer-title h5,
+.article-footer-title a {
+  color: rgb(138, 134, 134);
+  font-size: 12px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: normal;
+}
+
+/* ---------------------------Recommended Articles------------------------ */
+
+.recommended-articles-containner hr {
+  width: 770px;
+  height: 2px;
+  margin: 0 5px 20px 31px;
+  background-color: black;
+}
+
+.more {
+  display: flex;
+  flex-wrap: wrap;
+  grid-column-start: 1;
+  grid-column-end: 2;
+  grid-row-start: 1;
+  grid-row-end: 2;
+  justify-content: space-evenly;
+}
+
+.more-title {
+  text-align: start;
+  padding-left: 32px;
+}
+
+.more-title a {
+  text-decoration: none;
+  color: black;
+}
+
+.more-title h4 {
+  padding: 0 0 20px 0;
+}
+
+.img-conatiner img {
+  width: 232px;
+  height: 181px;
+}
+
+.img-conatiner h6 {
+  text-align: end;
+  color: gray;
+  font-weight: normal;
+}
+
+.img-description {
+  text-align: start;
+}
+
+.img-description h6 {
+  color: grey;
+  font-weight: normal;
+}
+
+.more-article-container {
+  width: 232px;
+  margin: 5px;
+  height: calc(50% - 10px);
+}
+
+/* -----------------Most popular list---------------------------------- */
+
+.most-popular {
+  grid-column-start: 2;
+  grid-column-end: 3;
+  grid-row-start: 1;
+  grid-row-end: 3;
+}
+
+.most-popular hr {
+  width: 350px;
+  height: 2px;
+  margin: 0 5px 20px 0;
+  background-color: black;
+}
+
+.most-popular-container {
+  text-align: start;
+  padding: 10px;
+  border-top: 1px lightgray solid;
+}
+
+.most-popular h3 {
+  padding: 0 0 20px 0;
+}
+
+.most-popular-container a {
+  text-decoration: none;
+  color: black;
+}
+
+.most-popular-container p {
+  font-size: 20px;
+  font-family: 'Times New Roman', Times, serif;
+  font-weight: 500;
+}
+
+/* ------------------------Editor's Picks---------------------- */
+
+.Editors-picks {
+  grid-column-start: 1;
+  grid-column-end: 2;
+  grid-row-start: 2;
+  grid-row-end: 3;
+}
+
+.Editors-picks hr {
+  width: 770px;
+  height: 2px;
+  margin: 0 5px 20px 31px;
+  background-color: black;
+}
+
+.Editors-picks-title {
+  text-align: start;
+  padding-left: 32px;
+  padding-bottom: 20px;
+}
+
+.Editors-picks-videos {
+  display: flex;
+  justify-content: space-evenly;
+  height: 90%;
+}
+
+.editors-videos {
+  width: 232px;
+}
+
+/* ------------------------Footer------------------------------ */
+
+.nav-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin: 0 0 10px 0;
+}
+
+.footer-title {
+  display: flex;
+  flex-direction: row;
```

Comment:
## No repeating of the same rule[1]

Try this:
```
.flex{
 display:flex;
}
```

In your HTML elements

`<div class="flex">`

- [ ] Delete all `display:flex;` rule you have in this file and write it the class on your HTML file

###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_